File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0103U12MakeSangyongTabInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OCS0103U12MakeSangyongTabInfo
	{
		private String _orderGubun;
		private String _orderGubunName;
		private String _memb;
		private String _ynValue;
		private String _sort;

		public String OrderGubun
		{
			get { return this._orderGubun; }
			set { this._orderGubun = value; }
		}

		public String OrderGubunName
		{
			get { return this._orderGubunName; }
			set { this._orderGubunName = value; }
		}

		public String Memb
		{
			get { return this._memb; }
			set { this._memb = value; }
		}

		public String YnValue
		{
			get { return this._ynValue; }
			set { this._ynValue = value; }
		}

		public String Sort
		{
			get { return this._sort; }
			set { this._sort = value; }
		}

		public OCS0103U12MakeSangyongTabInfo() { }

		public OCS0103U12MakeSangyongTabInfo(String orderGubun, String orderGubunName, String memb, String ynValue, String sort)
		{
			this._orderGubun = orderGubun;
			this._orderGubunName = orderGubunName;
			this._memb = memb;
			this._ynValue = ynValue;
			this._sort = sort;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0103U12MakeSangyongTabInfo.java
package nta.med.data.model.ihis.ocsa;

public class OCS0103U12MakeSangyongTabInfo {
	private String orderGubun;
	private String orderGubunName;
	private String memb;
	private String ynValue;
	private String sort;
	public OCS0103U12MakeSangyongTabInfo(String orderGubun,
			String orderGubunName, String memb, String ynValue, String sort) {
		super();
		this.orderGubun = orderGubun;
		this.orderGubunName = orderGubunName;
		this.memb = memb;
		this.ynValue = ynValue;
		this.sort = sort;
	}
	public String getOrderGubun() {
		return orderGubun;
	}
	public void setOrderGubun(String orderGubun) {
		this.orderGubun = orderGubun;
	}
	public String getOrderGubunName() {
		return orderGubunName;
	}
	public void setOrderGubunName(String orderGubunName) {
		this.orderGubunName = orderGubunName;
	}
	public String getMemb() {
		return memb;
	}
	public void setMemb(String memb) {
		this.memb = memb;
	}
	public String getYnValue() {
		return ynValue;
	}
	public void setYnValue(String ynValue) {
		this.ynValue = ynValue;
	}
	public String getSort() {
		return sort;
	}
	public void setSort(String sort) {
		this.sort = sort;
	}
}
